python/src/opendp/_convert.py

```diff
@@ -36,18 +36,24 @@ def c_int_limits(type_name):
 }
 _ERROR_URL_298 = "https://github.com/opendp/opendp/discussions/298"
 
-def check_similar_scalar(expected, value):
+
+def check_and_cast_scalar(expected, value):
```

Comment:
@mccalluc : based on your last comment, can add a docstring to this function. Thanks

Reply:
will do.